profiles/default/bsd/package.use

```diff
@@ -0,0 +1,8 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+# Michał Górny <mgorny@gentoo.org> (26 Jan 2017)
+# Preserve the old defaults on *BSD systems.
+sys-libs/libcxx -libcxxabi libcxxrt -libunwind
+sys-libs/libcxxrt -libunwind
```

Comment:
@aballier,
At least, I think we need to add fbsd keywords to sys-libs/libcxxabi and sys-libs/llvm-libunwind.
Both packages need to force ABI_X86="32 64" on amd64-fbsd.

<=sys-libs/llvm-libunwind-3.9.1 requires 32 bits of llvm.
@mgorny said that it was fixed in 4.0.0_alpha1.
https://bugs.gentoo.org/show_bug.cgi?id=607390

I'll test them after llvm-4.0.0 has been released.
And if I can not find a problem I'll open a PR for the default change.